### ISSUE TITLE
When people have "loggedin" 

### ISSUE THREAD

**Issue by steffanjensen** [gexxxter/coinHiveMiner#4]
When people have "loggedin" how do i output the id from the database into a text. Lets say "Welcome back id "x"" ?

thanks awesome script

**Comment by gexxxter** [comment]
Do you want to output the numeric id from the database i.e. 1337  or the username like "Hi SlaxXx" ?

**Comment by steffanjensen** [comment]
Like hi 1337

**Comment by gexxxter** [comment]
With commit https://github.com/gexxxter/coinHiveMiner/commit/c84bb696238c6cadcb163a07808b5bb9e63328b1
I changed the loginUser.php to now return the user id as json.

To access and use the id in miner.js change the call to api/loginUser.php and interpret the response instead of ignoring it like I did.

To do so do the same changes to your local files like I did in this commit.
https://github.com/gexxxter/coinHiveMiner/commit/eb97287845a738644eca7fe5c85e80d9af8c25aa

Hope this helps.
